numba/unicode.py

```diff
@@ -1015,6 +1015,14 @@ def wrap(a, b):
         return wrap
 
 
+@overload(operator.not_)
+def unicode_not(a):
+    if isinstance(a, types.UnicodeType):
+        def impl(a):
+            return True if a == "" else False
```

Comment:
I think this can be just `len(a) == 0`.  I didn't look at the optimized LLVM but I think this will result in too much operations for the simple check.

Reply:
Good point. Fixed in f9a36af